src/week1.org

```diff
@@ -0,0 +1,41 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I?
+- Kristaps Berzinch
+  - RoboJackets Treasurer
+  - Linux Foundation Certified Engineer
+- How to contact me
+  - Slack: @kristapsberzinch
+  - Email: kristaps@robojackets.org
+
+* Why are you here?
+- Learn the basics of software development
+- Get up to speed on technologies and techniques common to most RoboJackets teams
+- TODO
+
+* The Linux terminal
+- Powerful text-based interface for interacting with your computer
+- (Almost) everything is a file
+- There's one big happy file system
+- Lots of tiny tools you can chain together to do useful things
+
+* Basic commands
+| Command | Use |
+|-------+------|
+| =cd= | Change to a different directory |
+| =ls=  | List files in this directory |
+| =mkdir= | Make a new directory |
+| =rm= | Remove a file |
+| =rmdir= | Remove a directory |
+| =cp= | Copy a file |
+| =mv= | Move a file |
+| =man= | Access documentation about a command |
+| =grep= | Search for a string |
```

Comment:
Also, FWIW, I would prefer a 'tutorial walk-through' instead of a simple table, which would go through all the commands and reach a clear goal that everyone can reach. I think retention is a lot better if people feel they've applied it than just look at a table. My [git slides](https://robojackets.github.io/robocup-training/slides/2016/git) are an example of this. I don't know if we have enough time to do this though, or if it's a good idea to do this for everyone (we'll have a lot of people at this point).